bootloader/bootloader.go

```diff
@@ -343,3 +343,40 @@ func ForGadget(gadgetDir, rootDir string, opts *Options) (Bootloader, error) {
 	}
 	return nil, ErrBootloader
 }
+
+// BootImage represents each image in the chains of trusted assets and kernels
+// used in the boot process. A boot image can be an EFI binary or a snap file
+// containing an EFI binary.
+type BootImage struct {
+	// Path is the path to the boot image file.
+	Path string
+	// Relative is the path to the EFI image inside a snap container.
+	Relative string
```

Comment:
I would still prefer these string to be called either:

Snap and Relative    (in the case of Snap == "", Relative would be the path)

another option is Path and Snap

Reply:
Will update accordingly.